bin/check-log.rb

```diff
@@ -128,6 +128,13 @@ class CheckLog < Sensu::Plugin::Check::CLI
          proc: proc(&:to_i),
          default: 250
 
+  option :encode_utf16,
```

Comment:
why not make the option just encoding? for example:
```ruby
option :encoding
        description: 'Encode line with the following encoding before matching',
        long: '--encoding $ENCODING'
        default: 'UTF-8'
```

This makes it easier when someone decides hey I want utf-32 encoding when that exists.